packages/gitbook/src/app/(site)/(content)/[[...pathname]]/not-found.tsx

```diff
@@ -1,11 +1,11 @@
 import { getSpaceLanguage, t } from '@/intl/server';
-import { getSiteLayoutData } from '@/lib/api';
+import { getCurrentSiteCustomization } from '@/lib/api';
 import { getSiteContentPointer } from '@/lib/pointer';
 import { tcls } from '@/lib/tailwind';
 
 export default async function NotFound() {
     const pointer = getSiteContentPointer();
-    const { customization } = await getSiteLayoutData(pointer);
+    const customization = await getCurrentSiteCustomization(pointer);
```

Comment:
`getCurrentSiteCustomization` only fetches the customization instead of the whole site data (for optimization)

Reply:
Although our middleware always fetches the complete `siteData` so maybe better to use that only 🤔 
https://github.com/GitbookIO/gitbook/blob/d22965dbe0d2602891777edf9cdc10d2412527fa/packages/gitbook/src/middleware.ts#L178-L184